src/main/java/duke/command/ListCommand.java

```diff
@@ -0,0 +1,24 @@
+package duke.command;
+
+import duke.storage.Storage;
+import duke.task.TaskList;
+
+/**
+ * Represents a command to list out all the tasks created by the user.
+ */
+public class ListCommand extends Command {
+    /**
+     * Lists out the tasks created by the user.
+     * @param tasks TaskList object which represents the list of tasks created by the user.
+     * @param storage storage which loads, saves and stores data of user.
+     * @return String to notify the user of the execution of this command.
+     */
+    public String execute(TaskList tasks, Storage storage) {
+        StringBuilder sb = new StringBuilder();
```

Comment:
The use of Stringbuilder increases efficiency when dealing with a long list of tasks, well done 👍